biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java

```diff
@@ -85,29 +86,39 @@
 		SYNTHETIC(ACC_SYNTHETIC),
 		MANDATED(ACC_MANDATED);
 
-		public static Access parse(String input) {
+		public static Set<Access> parse(String input) {
 			switch (input) {
 				case "OPEN" :
 				case "open" :
 				case "0x0020" :
 				case "32" :
-					return OPEN;
+					return Collections.singleton(OPEN);
 				case "SYNTHETIC" :
 				case "synthetic" :
 				case "0x1000" :
 				case "4096" :
-					return SYNTHETIC;
+					return Collections.singleton(SYNTHETIC);
 				case "MANDATED" :
 				case "mandated" :
 				case "0x8000" :
 				case "32768" :
-					return MANDATED;
+					return Collections.singleton(MANDATED);
 				default :
+					if (input.indexOf(',') > -1) {
+						return Strings.split(input)
+							.stream()
+							.map(Access::parse)
+							.flatMap(Set::stream)
+							.collect(toSet());
+					}
 					int parsedValue = Integer.decode(input);
+					if ((parsedValue != 0) && (parsedValue != ACC_MANDATED) && (parsedValue != ACC_OPEN)
+						&& (parsedValue != ACC_SYNTHETIC)) {
+						throw new IllegalArgumentException(input);
+					}
 					return Arrays.stream(values())
 						.filter(a -> a.getValue() == parsedValue)
-						.findFirst()
-						.orElseThrow(() -> new IllegalArgumentException(input));
+						.collect(toSet());
```

Comment:
```suggestion
						.findFirst()
						.map(EnumSet::of)
						.orElseThrow(() -> new IllegalArgumentException(input));
```

Then you don't need the testing for valid values above.